### PR TITLE
Add instructions how to integrate retext plugins

### PR DESCRIPTION
**Pull request by silvenon**

### CODE REVIEW

File: docs/advanced/retext-plugins.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Using Retext Plugins
+
+[Retext][] plugins are really useful, but MDX can't convert the syntax tree to retext because it wouldn't be possible to convert it back to remark. This means that we can't use retext plugins directly. [remarkjs/remark#224][]
+
+Luckily, it's possible to build a custom plugin that visits all text nodes using [unist-util-visit][] and processes them using retext:
```
[the remark this answers is not in the document]
Comment (silvenon) [reply docs/advanced/retext-plugins.md]: ✅

File: docs/advanced/retext-plugins.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Using Retext Plugins
+
+[Retext][] plugins are really useful, but MDX can't convert the syntax tree to retext because it wouldn't be possible to convert it back to remark. This means that we can't use retext plugins directly. [remarkjs/remark#224][]
```
Comment (timneutkens) [inline docs/advanced/retext-plugins.md]: Seems like the link is mising here 2 times

File: docs/advanced/retext-plugins.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+Luckily, it's possible to build a custom plugin that visits all text nodes using [unist-util-visit][] and process them using retext:
```
Comment (timneutkens) [inline docs/advanced/retext-plugins.md]: Same here, link missing

**Comment by timneutkens** [comment]
@silvenon just saw a few missing links 🙏

**Comment by johno** [comment]
They're there using reference-style link definitions 😸

**Comment by timneutkens** [comment]
Oh wow, I see, TIL 😄

**Pull request merged**